### PR TITLE
JNG-5128 Add instances for builders

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5128" title="JNG-5128" target="_blank">TaskJNG-5128</a>  SDK should keep the database through test cases
  </td></table>
  <br />

### CODE REVIEW

**Review by robertcsakany — CHANGES_REQUESTED** [review]
Do not change API. In that case create a JudoHsqldbTestModule because internal details bleeding out because of test context

File: judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java [hunk 1]
```diff
@@ -55,18 +65,25 @@ public static class JudoPostgresqlModulesBuilder {
...
 
     @Override
     protected void configureDialect() {
-        bind(Dialect.class).toInstance(new PostgresqlDialect());
+        if(dialect != null) {
+            bind(Dialect.class).toInstance(dialect);
+        }else {
+            bind(Dialect.class).toInstance(new PostgresqlDialect());
+        }
     }
```
Comment (robertcsakany) [inline judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java]: In postgreSql the dialect HAVE TO BE PostresqlDialect.

File: judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java [hunk 2]
```diff
@@ -55,18 +65,25 @@ public static class JudoPostgresqlModulesBuilder {
...
 
     @Builder
-    private JudoPostgresqlModules(String host, Integer port, String user, String password, String databaseName, Integer poolSize) {
+    private JudoPostgresqlModules(String host, Integer port, String user, String password, String databaseName, Integer poolSize, Dialect dialect, DataSource source, PostgresqlRdbmsInit postgresqlRdbmsInit) {
```
Comment (robertcsakany) [inline judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java]: source have to be datasource

**Review by robertcsakany — CHANGES_REQUESTED** [review]

**Review by noherczeg — APPROVED** [review]

File: judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java (hunk 2 above)
Comment (gaborflorian) [reply judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java]: [06d0feb](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/148/commits/06d0febfc2bca7ca64bda4d16aec4a828197c7c3)

File: judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java (hunk 1 above)
Comment (gaborflorian) [reply judo-runtime-core-bootstrap-postgresql/src/main/java/hu/blackbelt/judo/runtime/core/bootstrap/dao/rdbms/postgresql/JudoPostgresqlModules.java]: [06d0feb](https://github.com/BlackBeltTechnology/judo-runtime-core/pull/148/commits/06d0febfc2bca7ca64bda4d16aec4a828197c7c3)

**Review by robertcsakany — CHANGES_REQUESTED** [review]
Test modules have to be defined in test project

**Comment by gaborflorian** [comment]
Test modules moved to the test project
judo-runtime-core-jsl

**Pull request closed without merging**